src/tribler/core/components/libtorrent/tests/test_download_manager.py

```diff
@@ -256,15 +276,30 @@ async def test_start_download_existing_handle(fake_dlmgr):
     await download.shutdown()
 
 
+def test_convert_rate():
+    assert DownloadManager.convert_rate(0) == -1  # 0 is a special value
+    assert DownloadManager.convert_rate(-1) == 1  # -1 is a special value
+    assert DownloadManager.convert_rate(1) == 1024
+    assert DownloadManager.convert_rate(2) == 2048
+    assert DownloadManager.convert_rate(-2) == -2048
+
+
+def test_reverse_convert_rate():
+    assert DownloadManager.reverse_convert_rate(-1) == 0  # -1 is a special value
+    assert DownloadManager.reverse_convert_rate(1) == -1  # 1 is a special value
+    for test_num in [0, 2, -2]:
+        assert DownloadManager.reverse_convert_rate(test_num) == test_num // 1024
```

Comment:
There is a bug in this test. If you remove the loop and rewrite it with flat lines, you will see:

```python
def test_reverse_convert_rate():
    assert DownloadManager.reverse_convert_rate(-1) == 0  # -1 is a special value
    assert DownloadManager.reverse_convert_rate(1) == -1  # 1 is a special value

    assert DownloadManager.reverse_convert_rate(0) == 0
    assert DownloadManager.reverse_convert_rate(2) == 0
    assert DownloadManager.reverse_convert_rate(-2) == -1
```

So, please, use the previously suggested variant.

Reply:
I did locally. I must have pushed it wrong. Same for the message above. I'm not sure what I did wrong.

Reply:
No, wait, I did miss this one too. My bad.